Source/Core/GoogleEarthEnterpriseMetadata.js

```diff
@@ -396,5 +471,8624 @@ define([
         return that._url + 'flatfile?q2-0' + quadKey + '-q.' + version.toString();
     }
 
+    var dbRootParser;
+
+    function requestDbRoot(that) {
+        if (!defined(dbRootParser)) {
+            dbRootParser = createDbRootParser();
+        }
+
+        var url = that._url + 'dbRoot.v5?output=proto';
```

Comment:
Always use `joinUrls` instead of adding strings together, this way other query params and hashes are preserved.